collector-sqs/src/main/java/zipkin/collector/sqs/SQSCollector.java

```diff
@@ -80,11 +80,18 @@ public Builder credentialsProvider(AWSCredentialsProvider credentialsProvider) {
 
     /** Amount of time to wait for messages from SQS */
     public Builder waitTimeSeconds(int seconds) {
-      checkArgument(parallelism > 0 && parallelism < 21, "waitTimeSeconds");
+      checkArgument(seconds > 0 && seconds < 21, "waitTimeSeconds");
       this.waitTimeSeconds = seconds;
       return this;
     }
 
+    /** Maximum number of messages to retrieve from SQS */
```

Comment:
Its the maximum number of messages that the SQS client will try to retrieve from SQS for each call.  I found it defaults to 1 though it allows for up to 10.  Setting this to a higher number will reduce the overall number of API calls made.  I will try to reword the comment to make this more clear.

http://docs.aws.amazon.com/AWSJavaSDK/latest/javadoc/com/amazonaws/services/sqs/model/ReceiveMessageRequest.html#getMaxNumberOfMessages--